examples/sidra/package.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
Please follow the structure with `name`, `version`, `private`, & `lagon-dev` script